util/raclgen/lib.py

```diff
@@ -230,24 +230,23 @@ def policy_name_to_idx(policy_name: str) -> int:
         try:
             base = int(range['base'], 0)
             size = int(range['size'], 0)
-            mask = size - 1
-            if size <= 0 or size.bit_count() != 1 or base & mask:
+            if size <= 0 or base < 0:
                 raise ValueError
```

Comment:
I realise that this isn't new, but I think it would probably be a good idea to add a string to the error to explain what has gone wrong.